tests/test_web_app_handler.py

```diff
@@ -109,9 +110,13 @@ def test_invalid_signature(monkeypatch, webhook_event):
 def test_handle_event_with_secret(monkeypatch, webhook_event):
     monkeypatch.setenv("GH_WEBHOOK_SECRET", "fake_secret")
     handler = WebAppHandler(dry_run=True)
+    handler.gh_client.send_trace = MagicMock()
     reason, http_code = handler.handle_event(
         data=webhook_event["payload"],
         headers=webhook_event["headers"],
     )
+    handler.gh_client.send_trace.assert_called_with(
```

Comment:
I don't know how this test passed in the past. It definitely should have been needing mocking responses since we should have been hitting Github's API.